docs/content/installation.md

```diff
@@ -232,13 +232,19 @@ Now download the image in that directory.
 
 ```bash
 cd /var/lib/wakame-vdc/images
-sudo curl -O http://dlc.wakame.axsh.jp.s3.amazonaws.com/demo/vmimage/ubuntu-lucid-kvm-md-32.raw.gz
+sudo curl -O https://dl.dropboxusercontent.com/s/chrx5fs1i3vi03t/ubuntu-14.04.3-x86_64-30g-passwd-login-enabled.raw.tgz?dl=0
```

Comment:
This command also adds the `?dl=0` to the filename. I think you'll have to specifically tell curl what filename to save it as.

HVA crashed with this in the log.

```
16:09:44] centos@joske  (/opt/axsh/wakame-vdc/frontend/dcmgr_gui) > tail -n 45 /var/log/wakame-vdc/hva.log                                                                                            
2015-10-19 16:06:01 JobContext thr=LocalStore[0/2] [ERROR]: Caught Dcmgr::Helpers::CliHelper::ShellRunner::CommandError: Unexpected exit code=2: cat /var/lib/wakame-vdc/images/ubuntu-14.04.3-x86_64-30g-passwd-login-enabled.raw.tgz| pv -W -f -p -s 312530432 | tar -zxS -C /var/lib/wakame-vdc/instances/i-pfcbxnq6/d20151019-2425-tsercf
Command PID: 2598
##STDERR=>
cat: /var/lib/wakame-vdc/images/ubuntu-14.04.3-x86_64-30g-passwd-login-enabled.raw.tgz: No such file or directory


gzip: stdin: unexpected end of file
tar: Child returned status 1
tar: Error is not recoverable: exiting now
        /opt/axsh/wakame-vdc/dcmgr/lib/dcmgr/helpers/cli_helper.rb:191:in `block in run!'
        /opt/axsh/wakame-vdc/dcmgr/lib/dcmgr/helpers/cli_helper.rb:189:in `tap'
        /opt/axsh/wakame-vdc/dcmgr/lib/dcmgr/helpers/cli_helper.rb:189:in `run!'
        /opt/axsh/wakame-vdc/dcmgr/lib/dcmgr/drivers/local_store/linux_local_store.rb:83:in `block in deploy_volume'
```

The `?dl=0` at the end of the filename caused the problem.

```
[16:11:01] centos@joske  (/opt/axsh/wakame-vdc/frontend/dcmgr_gui) > ls /var/lib/wakame-vdc/images/
ubuntu-14.04.3-x86_64-30g-passwd-login-enabled.raw.tgz?dl=0
```